Documentation/ExtensionArchitecture/PublishExtension/Index.rst

```diff
@@ -0,0 +1,93 @@
+.. include:: ../../Includes.txt
+
+
+.. _publish-extension:
+
+======================
+Publish Your Extension
+======================
+
+By publishing an extension, we mean making it publicly available. This may include
+all of or just a few of the following:
```

Comment:
I'd suggest to strongly recommend doing all of the following. Yes, sometimes there are strong reasons against one of these, but for a normal extension, all of that should be done.